remotes/docker/resolver.go

```diff
@@ -498,7 +499,7 @@ func (r *dockerBase) fetchTokenWithOAuth(ctx context.Context, to tokenOptions) (
 	if (resp.StatusCode == 405 && r.username != "") || resp.StatusCode == 404 {
 		return r.getToken(ctx, to)
 	} else if resp.StatusCode < 200 || resp.StatusCode >= 400 {
-		b, _ := ioutil.ReadAll(resp.Body)
+		b, _ := ioutil.ReadAll(io.LimitReader(resp.Body, 1e6))
```

Comment:
Just guessing on a size here.

Reply:
Let's say the absolute max size for a token is 16kb, we 3 tokens and 2 timestamps, 64KB seems like a reasonable limit since we do not define arbitrary data in the specification. Of course as soon as we define anything it will break something though :trollface: